### ISSUE TITLE
Decide which tools to use for planning

### ISSUE THREAD

**Issue by marigerr** [chingu-voyage-turtles-team-5/momentum-clone#1]
for example : Waffle.io, Trello, Google Docs, ReadMe files on Github

**Comment by marigerr** [comment]
If we decide use Waffle.io.   Does everyone like the pre-built columns?   We can customize the names and order.   I'm believe Backlog is for Issues that are still being discussed and unassigned.   Ready is for issues that are to ready to be worked on (but still unassigned?).   In progress is self explanatory - assigned/in progress work.    Pull Requests also get put into In progress column attached to the issue if included if "closes issue ###" is included in PR description.

**Comment by jrpcoder** [comment]
Yes, we can let it stay as it is. We may change it in future if needed.

**Comment by marigerr** [comment]
When researching about tracking multiple related issues, I came across another GitHub integration - [Zenhub ](https://github.com/marketplace/zenhub )that offers "epics" tracking.   Epics are a sort of parent for related issues.   Another thing neat about Zenhub is that it is integrated directly into Github by way of a Chrome extension, so it's like your working in an enhanced version of github, rather than going to a different platform like you do with Waffle.

We can stay with Waffle, but just thought I'd share what I found about Zenhub, since we are still early on in the project.  Zenhub just takes a couple clicks to enable.

Here are some screenshots from creating an epic from a personal repo of mine.

![image](https://user-images.githubusercontent.com/15229372/28744683-7901108e-7466-11e7-9d23-d5ecc67e136e.png)

![image](https://user-images.githubusercontent.com/15229372/28744741-b3e3fd82-7467-11e7-8b40-1b1872742b58.png)

![image](https://user-images.githubusercontent.com/15229372/28744759-0d493a72-7468-11e7-9bf7-6ec350fa4975.png)

### Screenshot of management board on github that Zenhub extension adds

![image](https://user-images.githubusercontent.com/15229372/28744779-7aa829ac-7468-11e7-88c5-333e6e6f5a34.png)

**Comment by marigerr** [comment]
I also think Jose's idea to have checklist in the parent/epic issues would work great without needing to switch to Zenhub!

**Comment by jrpcoder** [comment]
Ups, I missed your comments on this card :/ 
Zenhub looks great, but since we already made the majority of the cards, I think we can pass it for now.
Or were you considering Zenhub for additional features besides  the tasks?

**Comment by marigerr** [comment]
No problem!   It's kind of hard to see all the issue comments as they come in.  I wonder if there is a feature to review new comments across issues for one repo or waffle board.
I also edited this comment a couple times as I was thinking about it, so it changed from when I originally wrote the it.  :)
I was interested in Zenhub because they have they have also "Epics" tracking for issues that have several sub-issues.   But then I realized that maybe we didn't really have true "Epics" and that your way of tracking tasks in an issue with the checklist was great.  

Sounds good to stick with Waffle!
